docker/Dockerfile.notifications-aggregator.jvm

```diff
@@ -26,7 +26,7 @@ RUN microdnf install openssl curl ca-certificates ${JAVA_PACKAGE} \
     && chown 1001:root /deployments \
     && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
 
-ENV JAVA_OPTIONS="-Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:+ExitOnOutOfMemoryError"
+ENV JAVA_OPTIONS="-Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:+ExitOnOutOfMemoryError -Xlog:gc"
```

Comment:
I think this may not be what you want.
(I think) This flag makes the jvm print all(?) gc related events to stdout, not only the used collector

e.g.

```
❯ java -Xlog:gc target/notifications-common-1.0.0-SNAPSHOT-tests.jar
[0.014s][info][gc] Using G1
[0,738s][info][gc] GC(0) Pause Young (Normal) (G1 Evacuation Pause) 24M->2M(512M) 3,527ms
```

-XX:+PrintCommandLineFlags is probably what you are looking for

```
❯ java -XX:+PrintCommandLineFlags target/notifications-common-1.0.0-SNAPSHOT-tests.jar
-XX:G1ConcRefinementThreads=10 -XX:GCDrainStackTargetSize=64 -XX:InitialHeapSize=536870912 -XX:MaxHeapSize=8589934592 -XX:+PrintCommandLineFlags -XX:ReservedCodeCacheSize=251658240 -XX:+SegmentedCodeCache -XX:+UseCompressedClassPointers -XX:+UseCompressedOops -XX:+UseG1GC
```